src/path_handler.cpp

```diff
@@ -63,12 +63,12 @@ PathHandler::transformToGlobalPlanFrame(const geometry_msgs::msg::PoseStamped &
 nav_msgs::msg::Path PathHandler::transformPath(const geometry_msgs::msg::PoseStamped & robot_pose)
 {
   // Find relevent bounds of path to use
-  geometry_msgs::msg::PoseStamped global_pose = transformToGlobalPlanFrame(robot_pose);
+  geometry_msgs::msg::PoseStamped && global_pose = transformToGlobalPlanFrame(robot_pose);
```

Comment:
&& used without auto is rvalue, otherwise as i said it's like universal reference. Here i think its invalid coz global_pose has name